Mathlib/RingTheory/PowerSeries/Order.lean

```diff
@@ -317,34 +344,24 @@ theorem order_mul (φ ψ : R⟦X⟧) : order (φ * ψ) = order φ + order ψ :=
   simp only [order_eq_emultiplicity_X]
   exact emultiplicity_mul X_prime
 
--- Dividing `X` by the maximal power of `X` dividing it leaves `1`.
-@[simp]
-theorem divided_by_X_pow_order_of_X_eq_one : divided_by_X_pow_order X_ne_zero = (1 : R⟦X⟧) := by
-  rw [← mul_eq_left₀ X_ne_zero]
-  simpa using self_eq_X_pow_order_mul_divided_by_X_pow_order (@X_ne_zero R _ _)
-
--- Dividing a power series by the maximal power of `X` dividing it, respects multiplication.
```

Comment:
It seems that this comment has gotten lost! Let's preserve it, and I think even upgrade it to a docstring